packages/date-picker/src/DatePicker/DatePicker.spec.tsx

```diff
@@ -53,6 +52,10 @@ import { DatePicker } from '.';
  * Generally, this type of tests should _only_ exist in a test file for user-facing components.
  */
 
+const TEST_IDS = {
+  errorMessage: 'lg-form_field-feedback-error_message',
```

Comment:
could we export this from `form-field`?

Reply:
The idea of introducing a pattern of exporting internal test ids from each package seems iffy to me. Is there an internal directory we can rely on that doesn't involve exporting directly from the `form-field` package?

What do you think @bruugey @TheSonOfThomp ?

Reply:
i'd imagine that we would want to use this test id inside a test harness package. We're currently importing the test ids from form-field inside other packages.

Reply:
oh gotcha, I think I can use the one that's already being exported in place of this